### DISCUSSION TITLE
[ERROR] Duplicate users when migrating from LDAP to OIDC Authentication (Okta)

### DISCUSSION THREAD

**Discussion by stevbitmex** [quay/quay#2897] [Show and tell]
We're facing some issues when migrating from LDAP into OIDC (Okta). 

- Our legacy quay is using LDAP that has `test@example.com` is the user
After migrating to OIDC (Okta). When I tried to login with this user via OIDC (Okta App), it shows me errors (sees screenshot)
<img width="1081" alt="Screenshot 2024-05-21 at 13 52 59" src="https://github.com/quay/quay/assets/160202620/4ffc5106-ac90-42b9-a8c1-7d25afef261b">

- However, If I used a new okta users, then I can logged in successfully

Looking forward to your help on this matter
Thank you

**Comment by stevbitmex** [comment]
Found solution: https://access.redhat.com/solutions/7066153

**Comment by HammerMeetNail** [comment]
Solution identified, closing. Please reopen if still an issue.

**Comment by stevbitmex** [comment]
<img width="1654" alt="Screenshot 2024-05-22 at 22 50 30" src="https://github.com/quay/quay/assets/160202620/92b77b8c-5e47-4454-8a39-aa5912fa13fd">

I reopened this issues because I could not find any button to link OIDC External Logins (Okta):

- First, splunk running with LDAP authentication, I was able to login which is fine
- Then I changed authentication from LDAP => OIDC and I cannot login with same LDAP user

- Then, I have to switch back to original LDAP authentication & login as `superadmin`.
- Finally, I navigate to `External Logins` pannel as you can see above BUT I cannot find any button to link okta application as in docs: https://access.redhat.com/solutions/7066153

Could u help to guide me how can I solve or debug on this? 
Thank you
